src/main/java/com/example/carsharing/model/Car.java

```diff
@@ -0,0 +1,47 @@
+package com.example.carsharing.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@SQLDelete(sql = "UPDATE cars SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction("is_deleted = FALSE")
+@Table(name = "cars")
+@NoArgsConstructor
+@Data
+public class Car {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String brand;
+    @Column(nullable = false)
+    private String model;
+    @Column(nullable = false, unique = true)
+    @Enumerated(EnumType.STRING)
+    private CarType type;
+    @Column(nullable = false)
+    private int inventory;
+    @Column(name = "daily_fee", nullable = false)
+    private BigDecimal dailyFee;
+    @Column(name = "is_deleted", nullable = false)
+    private boolean isDeleted = false;
+
+    public enum CarType {
+        SEDAN,
+        SUV,
+        HATCHBACK,
+        UNIVERSAL
```

Comment:
it's our local name of car body type, in English it would be
```suggestion
        STATION_WAGON
```

Reply:
So change it on git with task
![image](https://github.com/RomanMenzheres/Car-Sharing/assets/118287818/b45b905f-2a4d-464e-8c5f-a476a9c40c14)